site/src/pages/TemplateSettingsPage/TemplateSettingsForm.tsx

```diff
@@ -55,11 +64,18 @@ export const TemplateSettingsForm: FC<TemplateSettingsForm> = ({
     initialValues: {
       name: template.name,
       description: template.description,
-      max_ttl_ms: template.max_ttl_ms,
+      // on display, convert from ms => hours
+      max_ttl_ms: template.max_ttl_ms / 3600000,
```

Comment:
nit: add comment explaining the 3600000 so future us remember what that represents